guidelines/README.md

```diff
@@ -0,0 +1,61 @@
+  - You don't need to have written it already, but have an idea of the points you want to make and how long it makes sense to talk about each one.
+- Include links to source code, blog posts, articles and other things that will add context to your talk.
+- Please feel free to thank our sponsors. Without them this wouldn't be possible!
+- Finally, if you want people to get in touch after your talk then let them know how to reach you. e.g: E-mail, Twitter, LinkedIn etc.
+
+With the good, there is always the bad. Bad ideas include:
+- Sales pitches/informercials.
+  - Feel free to talk about where you work. However, talks along the lines of "how to operate our product" is probably not a great idea.
+- How to operate your product. For example, I don't think anyone is interested in how to configure feature X on device Y. But an explanation of the internals should be fine.
+
+If you're not sure, you can always ask!
+
+## Panelists
+Ideally we'd like to spend a short amount of time talking to you before the panel. This is to get to know you and find out what your strengths are. This allows us to direct questions to the best person on the panel. We know you're busy so this is not always possible. If you could make time for a phone/Skype/FaceTime chat then that works fine too.
+Be prepared to introduce yourself to the audience. What should they know about you? What are you bringing to the discussion?
+In the interests of time we may not call upon you to answer a question. However, if you have a burning opinion on something and want to voice it during the panel then we'll figure something out.
+Also, opinionated folk tend to make for interesting discussions. However, remember, there's a human on the other side of the question!
+
+## Sponsors
+Sponsors get to announce their company, publicity via the meetup group, and an announcement from the compere during it.
+We have a few sponsorship options.
+- Social sponsorship.
+  This can take the form of a meet and greet. We're happy to announce the meeting using the Meetup group. It's then up to you to figure out what you want from it.
+- Venue sponsorship.
+  We're always on the lookout for new venues. If you've got something that can host a 100 people comfortably including food and drink, then we'd love to hear from you.
+- Video recording sponsorship.
+  You need to organise ecording and editing the event. You will need to supply the video files within a week of the event.
+
+## Attendees
+We'll usually wine and dine you which is not a bad deal for an evening of your time where you can learn and see some cool things!
+With this in mind, please be mindful of the following:
+- Please be curious and ask questions!
+- Be considerate of the other attendees. You're at this event because you're a professional. If it's something that won't fly in your workplace, then chances are it won't here either.
+That's it!
```

Comment:
Maybe add: clean up after yourself to help to get the venue back to the original state.